apps/meteor/tests/end-to-end/api/01-users.js

```diff
@@ -4229,4 +4229,179 @@ describe('[Users]', function () {
 			});
 		});
 	});
+
```

Comment:
Tests for the sendWelcomeEmail endp would be good too

Reply:
Done!